src/drivers/kinetis/uart.c

```diff
@@ -171,7 +171,7 @@ int uart_init(uart_config conf) {
 unsigned int uart_write(uart_context *context, char *buf, unsigned int bytes) {
     unsigned int bytes_written = 0;
     if(bytes == 0) {
-        bytes_written = context->txq->cap - context->txq->size;
+        /*bytes_written = context->txq->cap - context->txq->size;*/

```

Comment:
Should this be deleted?

Reply:
no - this is here because at one point it was useful for determining the number of bytes that could be written to the buffer.  `man 3 write` specifies that calling read/write with a zero number of bytes "may" return information about the number of bytes available.  I removed support for that because it was making it less convenient to block on write.  I'll add a comment saying what it does.